src/test/java/com/puppycrawl/tools/checkstyle/utils/ModuleReflectionUtilTest.java

```diff
@@ -75,6 +79,17 @@ public void testIsCheckstyleModule() {
                 .isTrue();
     }
 
+    @Test
+    public void testGetCheckstyleModules() throws IOException {
+        final ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+        final Set<String> packages =
+                Collections.singleton(BASE_PACKAGE + ".checks" + ".javadoc" + ".utils");
+
+        assertWithMessage("specified package has no checkstyle modules")
+                .that(ModuleReflectionUtil.getCheckstyleModules(packages, classLoader))
+                .isEmpty();
```

Comment:
Sorry we do not accept pure Junit test.
We need to cover by verifyxxxxx somehow.
If not possible, we need to write long explanation and put it over test method.

Reply:
@romani how am I supposed to cover this with verify? we aren't testing a check or a check property here how verify with input file can help?

Reply:
you skipping this conditions:
https://github.com/checkstyle/checkstyle/blob/a2d834fb1517cf9056dc70f4c8f1c4999d676f52/src/main/java/com/puppycrawl/tools/checkstyle/utils/ModuleReflectionUtil.java#L74-L78

can we make internal test module that do some violation or make it abstract class to let initialization to fail with exception.
Even easier: use XpathFileGenerator as module in Input file, and assert that it did not run.

Reply:
@romani sorry but please can you explain more what should I do. I think i am not understanding well

Reply:
@romani please check this https://github.com/checkstyle/checkstyle/pull/14499#issuecomment-1952291697, we are not even hitting our mutated function when using verify and input config. I have tried many other tests that use verify and we aren't hitting it.